src/io/sampling.jl

```diff
@@ -435,9 +365,9 @@ function fix_factor!(d::Domain, factor::Symbol)::Nothing
     params = DataFrame(d.model)
     default_val = params[params.fieldname .== factor, :val][1]
 
-    bnds = params[params.fieldname .== factor, :bounds][1]
+    bnds = params[params.fieldname .== factor, :dist_params][1]
```

Comment:
You probably won't want to handle that now, but this variable could be renamed - probably to `dist_params`.